model/src/main/kotlin/utils/VersionUtils.kt

```diff
@@ -42,9 +42,9 @@ internal fun Identifier.isApplicableIvyVersion(pkgId: Identifier) =
 
         // `Semver.satisfies(String)` requires a valid version range to work as expected, see:
         // https://github.com/semver4j/semver4j/issues/132.
-        val range = getVersionRange() ?: return false
+        val rangesList = getVersionRange() ?: return false
```

Comment:
IMO "rangesList" sounds a bit bulky, even if the original type is called so. COuld we simply call this "ranges" (plural)? And should "getVersionRange" also be renamed to "getVersionRanges" (plural) then?

Reply:
I wanted to make the naming consistent with the one in `isVersionRange` and this one already has a variable named `ranges`.

Reply:
I've found a way to address both by inlining one of the variables, see https://github.com/oss-review-toolkit/ort/pull/10440.